spyder/plugins/editor.py

```diff
@@ -976,7 +986,7 @@ def get_plugin_actions(self):
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 
         file_menu_actions = [self.new_action,
-                             None,
+                             self.open_last_closed,

```

Comment:
Don't remove `None` here and move `self.open_last_closed` after `self.open_action`.

We use `None` to create separators in our menus :-)